.github/workflows/create-release.yml

```diff
@@ -0,0 +1,28 @@
+# Push image to GHCR and Helm Chart to helm-charts-registry
+name: Release Image and Helm Chart
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+concurrency:
+  group: dbildungs-iam-server-image-helm-chart
```

Comment:
I think it should be schulportal-client instead of dbildungs-iam-server here, right?